modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -402,6 +413,19 @@ public String getDownloadFileEntryURL(
 		return cpMedia.getDownloadURL();
 	}
 
+	@Override
+	public CommerceInventoryReplenishmentItem getFirstOrderedIncomingQuantity(
+		long companyId, String sku) {
+
+		OrderByComparator<CommerceInventoryReplenishmentItem>
+			orderByComparator =
+				new CommerceReplenishmentAvailabilityDateComparator(true);
```

Comment:
Remove this variable and do it inline. Rename `CommerceReplenishmentAvailabilityDateComparator` to `CommerceInventoryReplenishmentItemAvailabilityDateComparator`. No need to pass params here, it is `true` by default.

Reply:
That class was already there, do you want me to rename it anyway?